packages/astro/src/core/build/internal.ts

```diff
@@ -84,13 +84,10 @@ export interface BuildInternals {
 	// A list of all statics chunks and assets that are built in the client
 	clientChunksAndAssets: Set<string>;
 
-	// The SSR entry chunk. Kept in internals to share between ssr/client build steps
-	ssrEntryChunk?: Rollup.OutputChunk;
 	manifestFileName?: string;
 	componentMetadata: SSRResult['componentMetadata'];
 	middlewareEntryPoint: URL | undefined;
 	astroActionsEntryPoint: URL | undefined;
-	serverIslandsEntryPoint: URL | undefined;
```

Comment:
Why was this removed? Shouldn't we remove the other entry points too?

Reply:
We no longer have separate entrypoints for each of these features. Yes agreed they should be removed, will do so in a future refactor.